R/navbar.R

```diff
@@ -30,7 +30,7 @@ data_navbar <- function(pkg = ".", depth = 0L) {
 
   list(
     type = navbar$type %||% "light",
-    bg = navbar$bg %||% "light",
+    bg = navbar$bg %||% pkg$meta$template$bslib$`navbar-bg` %||% "light",
```

Comment:
Should this completely get rid of `navbar$bg`?

Reply:
(& update of the related docs)

Reply:
I have no sense of the pros and cons of the options here.

Reply:
A pro of keeping `navbar$bg` (would need some fixes as I see now) is that one could suggest users only use `template$bootswatch`, `navbar$type` & `navbar$bg` to begin with. @apreshill what do you think?

Reply:
Removing the non-bslib option after hearing @apreshill's opinion elsewhere.